velodyne_pointcloud/tests/test_calibration.cpp

```diff
@@ -146,44 +145,3 @@ TEST(Calibration, hdl64e_s21)
   EXPECT_EQ(laser.max_intensity, 255);
   EXPECT_EQ(laser.min_intensity, 0);
 }
-
-TEST(Calibration, hdl64e_s2_float_intensities)
-{
-  std::string g_package_path = ros::package::getPath("velodyne_pointcloud");
-  Calibration calibration(
-    g_package_path + "/tests/issue_84_float_intensities.yaml", false);
-  EXPECT_TRUE(calibration.initialized);
-  ASSERT_EQ(calibration.num_lasers, 64);
-
-  // check some values for the first laser:
-  LaserCorrection laser = calibration.laser_corrections[0];
-  EXPECT_FALSE(laser.two_pt_correction_available);
-  EXPECT_FLOAT_EQ(laser.vert_correction, -0.12118950050089745);
-  EXPECT_FLOAT_EQ(laser.horiz_offset_correction, 0.025999999);
-  EXPECT_EQ(laser.max_intensity, 255);
-  EXPECT_EQ(laser.min_intensity, 40);
-
-  // check similar values for laser 26:
-  laser = calibration.laser_corrections[26];
-  EXPECT_FALSE(laser.two_pt_correction_available);
-  EXPECT_FLOAT_EQ(laser.vert_correction, -0.014916840599137901);
-  EXPECT_FLOAT_EQ(laser.horiz_offset_correction, 0.025999999);
-  EXPECT_EQ(laser.max_intensity, 245);
-  EXPECT_EQ(laser.min_intensity, 0);
-
-  // check similar values for the last laser:
-  laser = calibration.laser_corrections[63];
-  EXPECT_FALSE(laser.two_pt_correction_available);
-  EXPECT_FLOAT_EQ(laser.vert_correction, -0.20683046990039078);
-  EXPECT_FLOAT_EQ(laser.horiz_offset_correction, -0.025999999);
-  EXPECT_EQ(laser.max_intensity, 255);
-  EXPECT_EQ(laser.min_intensity, 0);
-}
```

Comment:
Why remove this test?

Reply:
I restore the test in https://github.com/ros-drivers/velodyne/pull/452/commits/96707883e0632b1fd278efe712b070ed6e3767c2.